src/codeu/chat/client/core/Context.java

```diff
@@ -22,15 +22,23 @@
 import codeu.chat.common.User;
 import codeu.chat.util.Uuid;
 import codeu.chat.util.connections.ConnectionSource;
+import codeu.chat.common.ServerInfo;
+import codeu.chat.client.core.View;
 
 public final class Context {
 
   private final BasicView view;
+  // We need to access the information of the view, but since 
+  // the method that returns this is private in the child class 
+  // BasicView, we're going to create a variable ViewInterface,
+  // which has a public version of this function - Priyanka Agarwal
+  private final View viewInterface;
   private final Controller controller;
 
   public Context(ConnectionSource source) {
     this.view = new View(source);
     this.controller = new Controller(source);
+    this.viewInterface = new View(source);
```

Comment:
I removed the viewInterface object and have made everything into a object called view of type View